metadata-ingestion/examples/library/lineage_emitter_dataset_finegrained_sample.py

```diff
@@ -0,0 +1,53 @@
+import datahub.emitter.mce_builder as builder
+from datahub.emitter.mcp import MetadataChangeProposalWrapper
+from datahub.emitter.rest_emitter import DatahubRestEmitter
+from datahub.metadata.com.linkedin.pegasus2avro.dataset import (
+    DatasetLineageType,
+    FineGrainedLineage,
+    FineGrainedLineageDownstreamType,
+    FineGrainedLineageUpstreamType,
+    Upstream,
+    UpstreamLineage,
+)
+
+
+def datasetUrn(tbl):
+    return builder.make_dataset_urn("hive", tbl)
+
+
+def fldUrn(tbl, fld):
+    return builder.make_schema_field_urn(datasetUrn(tbl), fld)
+
+
+fineGrainedLineages = [
+    FineGrainedLineage(
+        upstreamType=FineGrainedLineageUpstreamType.FIELD_SET,
+        upstreams=[
+            fldUrn("fct_users_deleted", "browser_id"),
```

Comment:
this doesn't look right -> does this first parameter not need to be a full dataset urn with platform, etc?

Reply:
Ah I see the util's make hive implicit- got it!